extras/package_gd32_index.json.tpl

```diff
@@ -28,17 +28,17 @@
           "name": "GigaDevice GD32",
           "toolsDependencies": [
             {
-              "packager": "GD32Community",
+              "packager": "GD32Community-fork",
               "name": "xpack-arm-none-eabi-gcc",
               "version": "9.3.1-1.3"
             },
             {
-              "packager": "GD32Community",
+              "packager": "GD32Community-fork",
               "name": "xpack-openocd",
-              "version": "0.11.0-1"
+              "version": "0.11.0-4"
             },
             {
-              "packager": "GD32Community",
+              "packager": "GD32Community-fork",
```

Comment:
For the merge, this should be reverted to the original one.

Reply:
```suggestion
              "packager": "GD32Community",
```